fluent-plugin-elasticsearch-timestamp-check.gemspec

```diff
@@ -13,5 +13,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_runtime_dependency "fluentd", [">= 0.14.0", "< 2"]
```

Comment:
do we want to just use '~> 0.14' instead?

Reply:
Fluentd v0.14 is planned to bump up version to v1 with the same functionality.
If you use `~> 0.14`, it cannot handle this bumping up.

Reply:
fair point